src/sql/src/session/vars.rs

```diff
@@ -404,6 +404,21 @@ const TRANSACTION_ISOLATION: ServerVar<IsolationLevel> = ServerVar {
     internal: false,
 };
 
+pub const MAX_KAFKA_CONNECTIONS: ServerVar<u32> = ServerVar {
+    name: UncasedStr::new("max_kafka_connections"),
+    value: &0,
+    description:
+        "The maximum number of Kafka connections in the region, across all schemas (Materialize).",
+    internal: false,
+};
+
+pub const MAX_POSTGRES_CONNECTIONS: ServerVar<u32> = ServerVar {
+    name: UncasedStr::new("max_postgres_connections"),
+    value: &0,
```

Comment:
do you want these to default to zero?

Reply:
I'm not sure how we want to go about this. I think @benesch has a concrete plan in mind so let's see what he says

Reply:
I would default them to 1000 (to match `max_connections`) here, and then you won't need to do anything special in CI/tests to bump them to 9001; the default of 1000 should be sufficient.

We'll use LD to scope this down for sandbox trials.